guava/src/com/google/common/hash/BloomFilter.java

```diff
@@ -337,7 +343,7 @@ public int hashCode() {
    * @since 23.0
    */
   public static <T> Collector<T, ?, BloomFilter<T>> toBloomFilter(
-      Funnel<? super T> funnel, long expectedInsertions, double fpp) {
+      Funnel<? super T> funnel, @Positive long expectedInsertions, @NonNegative @LessThan("1") double fpp) {
```

Comment:
`LessThan` is supposed to apply to integral types, not floating point types. I think we should remove all of these `LessThan` annotations, because they'll cause false positives anytime these methods are used (and don't contribute to typechecking this class). For example, the warning suppressions at lines 320, 486, and 532 are caused by these.